cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -93,6 +93,7 @@ func (p *nfsProvisioner) Provision(ctx context.Context, options controller.Provi
 		data: map[string]string{
 			"name":      pvcName,
 			"namespace": pvcNamespace,
+			"volume":    options.PVName,
```

Comment:
btw, why `volume` and not `pvname`?

Reply:
no real reason, mainly because binds it to a volume that is created by the storage manager. I'm open to change it to pvname since that clearly indicates that that part comes from the persistent volume name.